packages/ipfs-cli/test/pubsub.spec.js

```diff
@@ -101,19 +101,19 @@ describe('pubsub', () => {
     }
 
     it('should publish message', async () => {
-      const subName = 'sub-name'
-      const data = 'data'
+      const subName = 'sub-name-1'
+      const data = 'data\r\nfirst\nZażółć gęślą jaźń😇'
```

Comment:
Yellow goose self?

Reply:
Sounds better in Polish ;) (it is a valid sentence that includes all polish diacritics, good for testing useful utf8)